state/state.go

```diff
@@ -216,7 +217,11 @@ type GrantS3 struct {
 	S3Fields map[string]string `json:"s3_fields"`
 }
 
+var ErrReplicaCollectionDisabled error = errors.New("monitored server is replica and replication collection disabled via config")
```

Comment:
Nice - good idea to keep errors in a central place. I wonder if we should even introduce a new `state/errors.go` file for this?